packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -482,15 +493,31 @@ export abstract class SharedObject<TEvent extends ISharedObjectEvents = ISharedO
     /**
      * {@inheritDoc @fluidframework/datastore-definitions#(IChannel:interface).getAttachSummary}
      */
-    public getAttachSummary(fullTree: boolean = false, trackState: boolean = false): ISummaryTreeWithStats {
-        return this.summarizeCore(this.serializer);
+    public getAttachSummary(
```

Comment:
I'm not sure how much we care about getAttachSummary() flow - it's used only when file is created (for the first time) and when new DDS is created. It's Ok to have it (and probably is a plus), just mentioning in case it causes trouble :)